### ISSUE TITLE
Test Case: Improved background location performance for Android

### ISSUE THREAD

**Issue by anshuman-mathur** [anshuman-mathur/AppStudio-Testing#3]

**Comment by paul-gpsit** [comment]
Is that Background_Location.txt example part of a larger app? It references some components such as MainPage, MapPage, PositionLog etc. Are these part of an existing template or sample and available?

**Comment by anshuman-mathur** [comment]
@paul-gpsit yes you are right. What I can do is give you a zip of the entire test app (except the iteminfo.json) which you can use to test.

[Background Location.zip](https://github.com/anshuman-mathur/AppStudio-Testing/files/2296023/Background.Location.zip)

**Comment by paul-gpsit** [comment]
Created app and tested in AppPlayer on Android Samsung Tab2 tablet. Worked fine, I enabled console logging and was able to see position updates continuing even when the app was in the background.

![image](https://user-images.githubusercontent.com/14798260/44249079-67558800-a242-11e8-9457-849fa14413a6.png)

**Comment by paul-gpsit** [comment]
Built app for Android and tested on Android Samsung Tab2 tablet. I had added console logging to the app so could see position updates coming through when the app was in the background.
So all looks fine to me.
